server/chat-plugins/friends.ts

```diff
@@ -585,6 +585,18 @@ export const pages: Chat.PageTable = {
 			buf += `value="/friends sharebattles off">Disable</button> `;
 			buf += `<button class="button${settings.displayBattlesToFriends ? ` disabled` : ``}" name="send" `;
 			buf += `value="/friends sharebattles on">Enable</button> <br /><br />`;
+
+			buf += `<strong>Block PMs except from friends (and Staff):</strong><br />`;
+			buf += `<button class="button${settings.blockPMs ? `` : ' disabled'}" name="send" `;
+			buf += `value="/unblockpms">Disable</button> `;
```

Comment:
```suggestion
			buf += `value="/unblockpms&#10;/friends settings">Disable</button> `;
```
This will refresh the page after clicking, otherwise people will just spam it thinking it's not working.
Can you add this to the rest of the commands as well?

Reply:
sure!